src/pages/exportUsers.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+
+export default function ExportUsersPage() {
+  const [error, setError] = useState<string | null>(null);
+
+  const convertToCSV = (data: any[]) => {
+    if (data.length === 0) return '';
+
+    const headers = Object.keys(data[0]).join(',') + '\n';
+    const rows = data
+      .map((row) =>
+        Object.values(row)
+          .map((value) => `"${value}"`)
+          .join(',')
+      )
+      .join('\n');
```

Comment:
it would be nice to handle potential edge cases like commas within the values
```suggestion
      .map((row) =>
        Object.values(row)
          .map((value) => `"${String(value).replace(/"/g, '""')}"`) // Escape quotes inside values
          .join(',')
      )
```